### ISSUE TITLE
ayylien baton bypasses Reactive teleport armor

### ISSUE THREAD

**Issue by YakumoChen** [tgstation/-tg-station#14599]
Also ayylien grabs don't bypass it so you get spaced every time you get abducted as RD, BUT NOPE NONE OF THE 40000000 TIMES I GOT AYYLIEN BATON'D WILL HELP

**Comment by GunHog** [comment]
Hehehehehehe, So you were that pretty little blond RD we captured, complaining about how other other abdoctors ruined Xenobiology! I loved your response when you realized you were being abducted. I was the agent that captured you. My scientist was a bit of a newbie who did not know about stripping the armor or how to mouse drop you into the scanner. 

It is kind of silly, really. He was looting basically everyone, while I tried to return most items except an ID we got from the CE. I was actually trying to return your hand tele when the scientist triggered your teleportation and subsequent spacing. 

Addressing the issue report, this is entirely true. I hit her with my baton more than most Sec officers beat a clown!
